e2e/testdata/fn-render/fn-failure/.expected/config.yaml

```diff
@@ -15,3 +15,5 @@
 # One of the functions in the pipeline fails resulting in
 # non-zero exit code and no changes in the resources.
 exitCode: 1
+stdErr: "httpbin-gen:4:1: got newline, want primary expression"
+stdOut: "running function \"gcr.io/kpt-fn/starlark:unstable\": FAILED"
```

Comment:
@Shell32-Natsu @droot 

What's the rationale for having this part of stdout vs stderror? What's the combined output shown to the user?

Reply:
The combined output is as follows:
```
package ".": running function "gcr.io/kpt-fn/starlark:unstable": FAILED
fn.render: pkg .:
        pkg.render:
        pipeline.run: httpbin-gen:4:1: got newline, want primary expression 
```

The rationale is that stdout will have whether the function succeeded or failed, while stderr will have the error. 

In the case that a function succeeds, this stdout would be shown to the user: 
`package ".": running function "gcr.io/kpt-fn/starlark:unstable": SUCCESS`

Reply:
For the failure path, separating the debugging information between stdout and stderr seems like unnecessary complexity. Also note that we need to have a design on the desired UX tracked by https://github.com/GoogleContainerTools/kpt/issues/1557

Reply:
Also having separate stdout and stderror means, the final output can be interleaved differently. We actually want the exact aggregated output in the the order you have it there.